tests/unit/test_spaces.cpp

```diff
@@ -237,5 +289,4 @@ INSTANTIATE_TEST_SUITE_P(
                     std::make_pair(17, spaces_test::IP_dist_funcs_8ExtResiduals),
                     std::make_pair(7, spaces_test::L2_dist_funcs_2ExtResiduals),
                     std::make_pair(7, spaces_test::IP_dist_funcs_2ExtResiduals)));
-
-#endif // CPU_FEATURES_ARCH_X86_64
+#endif
```

Comment:
Let's avoid unrelated changes, so we have fewer conflicts in the future (at least, keep the new line at the end)